### PR TITLE
refactor: refactor and update util to use new log formatter setting

### PR DESCRIPTION
**Pull request by andy89923**
## Description
- Refactor logger to free5GC logger format
- Add CI actions 
- Update module package

### CODE REVIEW

File: pkg/radius/handler/3gpp_types.go [hunk 1]
```diff
@@ -252,10 +257,10 @@ func UnmarshalEAP5GData(codedData []byte) (eap5GMessageID uint8, anParameters *A
 
 							if len(parameterValue) < int(parameterLength) {
 								return 0, nil, nil, errors.New("Error formatting")
-							} else {
-								parameterValue = parameterValue[:parameterLength]
 							}
-
+							// else {
+							// 	parameterValue = parameterValue[:parameterLength]
+							// }
```
Comment (andy89923) [inline pkg/radius/handler/3gpp_types.go:263]: comment by mistake?

File: internal/ngap/service/service.go [hunk 2]
```diff
@@ -6,14 +6,14 @@ import (
 	"runtime/debug"
 	"time"
 
-	"git.cs.nctu.edu.tw/calee/sctp"
 	"github.com/sirupsen/logrus"
 
+	lib_ngap "github.com/free5gc/ngap"
```
Comment (andy89923) [inline internal/ngap/service/service.go:11]: could we not use an alias?
which means just 
```go
"github.com/free5gc/ngap"
```

File: pkg/radius/handler/3gpp_types.go (hunk 1 above)
Comment (blackcat-118) [reply pkg/radius/handler/3gpp_types.go:263]: parameterValue is unused variable in this case. Should I delete these code?

Comment (andy89923) [reply pkg/radius/handler/3gpp_types.go:263]: ok, deleted.

Comment (blackcat-118) [reply pkg/radius/handler/3gpp_types.go:263]: I have asked the senior and he said that I can ignore these codes.

File: internal/ngap/service/service.go (hunk 2 above)
Comment (blackcat-118) [reply internal/ngap/service/service.go:11]: line 14 also include tngf package named ngap. 

image

**Review by andy89923 — COMMENTED** [review]
LGTM

**Review by ianchen0119 — APPROVED** [review]
LGTM!

**Pull request merged**